app/api/business/evidence_business.py

```diff
@@ -45,3 +46,20 @@ def delete_draft_evidence(evidence_id, actioned_by):
         db_object=evidence
     )
     return True
+
+
+def get_domain_and_evidence_data(evidence_id):
```

Comment:
I did some testing and noticed that sellers may have categories that have been approved under the old assessment process.  We need to think about how we check for these approvals because currently no data is being returned for the front-end to display.  Have a look at [seller_dashboard.py](https://github.com/AusDTO/dto-digitalmarketplace-api/blob/master/app/api/views/seller_dashboard.py#L82) to see how old approvals are displayed in the dashboard.

Reply:
Resolved - added case studies view